solar/sankey_visualization.ipynb

```diff
@@ -0,0 +1,288 @@
+{
```

Comment:
288 lines of sankey ipynb on the wall, 288 lines of sankey,,,

Reply:
That's annoying.  How do I get rid of this?